### PR TITLE
[ISSUE #2680]Add namesrv startup process description

### PR DESCRIPTION
**Pull request by zsmweiwei**
**Make sure set the target branch to `develop`**

## What is the purpose of the change

XXXXX

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily. Notice, `it would be helpful if you could finish the following 5 checklist(the last one is not necessary)before request the community to review your PR`.

- [x] Make sure there is a [Github issue](https://github.com/apache/rocketmq/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [x] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [x] Write necessary unit-test(over 80% coverage) to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/apache/rocketmq/tree/master/test).
- [x] Run `mvn -B clean apache-rat:check findbugs:findbugs checkstyle:checkstyle` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

File: docs/cn/client/java/API_Reference_NamesrvStartup.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+|String|configStorePath|namesrv的配置文件路径，可以使用-c指定|
+|String|productEnvName|环境名|
+|boolean|clusterTest|是否开启集群测试，默认false|
+|boolean|orderMessageEnable|是否支持顺序消息，默认是false|
+
+#### nettyServerConfig参数说明
+|类型|参数名称|描述|
+|------|-------|-------|
+|int|listenPort|nameServer监听的端口，是8888，后面会被初始化为9876|
+|int|serverWorkerThreads|netty的线程池线程个数|
```
Comment (lizhiboo) [inline docs/cn/client/java/API_Reference_NamesrvStartup.md:50]: netty的woker线程数，处理namesrv的业务逻辑线程。

File: docs/cn/client/java/API_Reference_NamesrvStartup.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+1.调用start方法启动NamesrvController
+
+2.调用NamesrvController的initialize来初始化
+
+3.初始化过程如下：
+- 加载kv配置，存在一个kv配置管理器
+- 创建netty服务的对象，同时在onChannelDestroy方法规定了当broker宕机时要触发的动作，其实就是把registerBroker方法做的事回退掉
+- 创建netty处理响应的线程池，然后注册
+- 创建了一个定时线程池，调用scanNotActiveBroker方法每隔10s来检查哪些broker宕机了
```
Comment (lizhiboo) [inline docs/cn/client/java/API_Reference_NamesrvStartup.md:75]: 用于检查broker失去心跳

File: docs/cn/client/java/API_Reference_NamesrvStartup.md [hunk 3]
```diff
@@ -0,0 +1,172 @@
+##NamesrvStarup
```
Comment (ShannonDing) [inline docs/cn/client/java/API_Reference_NamesrvStartup.md:1]: 标题格式缺少空格

**Review by duhenglucky — APPROVED** [review]

**Comment by duhenglucky** [comment]
@zsmweiwei Would you like to modify these docs according to the comments？

**Comment by duhenglucky** [comment]
@zsmweiwei  ping

**Pull request closed without merging**